src/SD_binary/write_example.cpp

```diff
@@ -5,6 +5,10 @@
 #include <OneWire.h>
 #include <DallasTemperature.h>
 
+// I'm not editing this yet -- should be compared against tag last_jf_review,
+// but this code is not formatted, and I don't support editing this until I see
+// a flow-chart showing how the original code works
```

Comment:
I'm not editing/reviewing this file yet -- should be compared against tag last_jf_review, but this code is not formatted, and I don't support editing this until I see a flow-chart showing how the original code works